Resources/Prototypes/Entities/Markers/Spawners/mobs.yml

```diff
@@ -27,9 +27,9 @@
       - state: ai
   - type: ConditionalSpawner
     prototypes:
-      - MobCat
```

Comment:
This spawner looks like a duplicate of the one below on line 141. ID should be different and these mobs should stay as the default cats not the pets.

Reply:
this is fine since theres a generic cat spawner too